services/galley/src/Galley/Data.hs

```diff
@@ -460,6 +464,20 @@ defAccess One2OneConv (Just (Set []))     = [PrivateAccess]
 defAccess RegularConv (Just (Set []))     = [InviteAccess]
 defAccess _           (Just (Set (x:xs))) = x:xs
 
+
+maybeRole :: ConvType -> Maybe AccessRole -> AccessRole
+maybeRole SelfConv    Nothing    = privateRole
+maybeRole ConnectConv Nothing    = privateRole
+maybeRole One2OneConv Nothing    = privateRole
```

Comment:
I think since the only access role that makes sense for `Self, Connect and One2One` is `privateRole`, we should perhaps change this to: 
```
maybeRole SelfConv    _    = privateRole
maybeRole ConnectConv _    = privateRole
maybeRole One2OneConv _    = privateRole
```
to make sure we don't end up in a state where people can be added to such conversations?

Reply:
Yes, agreed.